map/manage-operational-layers/src/main/java/com/esri/samples/manage_operational_layers/ManageOperationalLayersSample.java

```diff
@@ -38,131 +38,142 @@
 import com.esri.arcgisruntime.mapping.LayerList;
+import com.esri.arcgisruntime.mapping.Viewpoint;
 import com.esri.arcgisruntime.mapping.view.MapView;
 
 public class ManageOperationalLayersSample extends Application {
 
+  private LayerList mapLayers;
   private MapView mapView;
-  private LayerList mapAddedLayers;
-
-  private static final String ELEVATION_LAYER =
-      "https://sampleserver5.arcgisonline.com/arcgis/rest/services/Elevation/WorldElevations/MapServer";
-  private static final String CENSUS_LAYER =
-      "https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer";
-  private static final String DAMAGE_LAYER =
-      "https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer";
 
   @Override
   public void start(Stage stage) {
 
     try {
-      // create stack pane and application scene
+      // create the stack pane and the application scene
       StackPane stackPane = new StackPane();
       Scene scene = new Scene(stackPane);
       scene.getStylesheets().add(getClass().getResource("/manage_operational_layers/style.css").toExternalForm());
 
-      // set title, size, and add scene to stage
+      // set a title, size, and add the scene to the stage
       stage.setTitle("Manage Operational Layers Sample");
       stage.setWidth(800);
       stage.setHeight(700);
       stage.setScene(scene);
       stage.show();
 
-      // create a control panel
-      VBox controlsVBox = new VBox(6);
-      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
-          Insets.EMPTY)));
-      controlsVBox.setPadding(new Insets(10.0));
-      controlsVBox.setMaxSize(200, 260);
-      controlsVBox.getStyleClass().add("panel-region");
+      // create an ArcGISMap with the topographic basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
 
-      // create labels for add/delete layers
-      Label addLayersLabel = new Label("Layers on the ArcGISMap");
-      Label deleteLayersLabel = new Label("Deleted Layers");
-      addLayersLabel.getStyleClass().add("panel-label");
-      deleteLayersLabel.getStyleClass().add("panel-label");
+      // create a view and add the ArcGISMap to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set the initial viewpoint for the map view
+      mapView.setViewpoint(new Viewpoint(37.056295, -111.195800, 10000000));
 
-      // create list for deleted layers
+      // add the image layers to the list of operational layers on the map
+      mapLayers = map.getOperationalLayers();
+      addMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Elevation/WorldElevations/MapServer");
+      addMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer");
+      addMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer");
```

Comment:
in reference to Jon's comment here, it's true that it's not really standard for Java samples to contain abstracted methods, such as here, for what is a pretty straightforward initialiser. Adding map image layers to the map's operational layers are a key part of this sample, so it's better to show the `ArcGISMapImageLayer`s instantiated right up front.

We can shorthand it a little with the following:
```suggestion
      // create and add new image layers to the map's operational layers
      LayerList mapLayers = map.getOperationalLayers();
      mapLayers.addAll(Arrays.asList(
        (new ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Elevation/WorldElevations/MapServer")),
        (new ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/DamageAssessment/MapServer")),
        (new ArcGISMapImageLayer("https://sampleserver5.arcgisonline.com/arcgis/rest/services/Census/MapServer"))));
```